include/ttmlir/Dialect/TTIR/IR/TTIROps.td

```diff
@@ -274,6 +274,13 @@ def TTIR_SqrtOp : TTIR_ElementwiseUnaryOp<"sqrt"> {
     }];
 }
 
+def TTIR_NegateOp: TTIR_ElementwiseUnaryOp<"neg"> {
```

Comment:
In ttnn and pytorch, the op is called `neg`. Let's change all the occurrences of `NegateOp` to `NegOp`.

Reply:
Thanks a lot. updated as suggested

Reply:
updated